architecture.md

```diff
@@ -644,13 +644,52 @@ a pod is **not** `Privileged` the runtime selection is done as follows:
 | Default CRI-O trust level: `untrusted` | Kata Containers                               | Kata Containers                              |  Kata Containers |
 
 
-### CRI-containerd
+### Containerd CRI Plugin (Formerly CRI-containerd)
+
+The general guidelines for the Containerd CRI Plugin support is similar to the CRI-O support. You can run trusted workloads with a runtime like `runc` and then run an untrusted workload with Kata Containers. The parameters that can be modified in the containerd config to run Kata Containers along with another 'trusted' runtime are the following:
```

Comment:
...The parameters that you can modify in the containerd...

Reply:
Added the change. Thx!